src/modules/CreateDrops/TicketDrop/CreateTicketDropSummary.tsx

```diff
@@ -0,0 +1,20 @@
+import { DropSummary } from '../DropSummary';
+
+import { useCreateTicketDropContext } from './CreateTicketDropContext/CreateTicketDropContext';
+
+export const CreateTicketDropSummary = () => {
+  const contextData = useCreateTicketDropContext();
+  const summaryData = contextData.getSummaryData();
```

Comment:
then, these will be 
``` const summaryData = getSummaryData();```

and so on

Reply:
or just passed `getSummaryData()` immediately ahhahha

Reply:
Refactored the code here [696872e](https://github.com/keypom/keypom-airfoil/pull/68/commits/696872e4f66e03e8ebb30490cc19138eb67947c1)